assets/components/Dropdown.tsx

```diff
@@ -85,8 +107,8 @@ Dropdown.propTypes = {
     reset: PropTypes.func,
     small: PropTypes.bool,
     stretch: PropTypes.bool,
-    buttonProps: PropTypes.shape({
-        textOnly: PropTypes.bool,
-    }),
     optionLabel: PropTypes.string,
+    borderless: PropTypes.bool,
+    hideLabel: PropTypes.bool,
+    hideLabelMobile: PropTypes.bool,
```

Comment:
are those actually set anywhere? I can't find any usage

Reply:
Not currently, but I had to add support for them because that are the requirements. So now we'll be using them sometime in the future I guess.